### DISCUSSION TITLE
data/dictionary_manual/*.txt in MacOS

### DISCUSSION THREAD

**Discussion by TanukiMa** [google/mozc#944] [General]
MacOS上でbuildすると
mozc/src/data/dictionary_manual/domain.txt, mozc/src/data/dictionary_manual/era.txt がシステム辞書としてbuildされていないようです。

githubから github actionsでBuildされた　Mozc.pkg をインストールしてみましたが、同様でした。

具体的には mozc/src/data/dictionary_manual/era.txt に記載されている「あたらしいげんごう」が「令和」に変換できません。

修正の予定はありますでしょうか？

**Comment by hiroyuki-komatsu** [comment]
Hi TanukiMa,

Thank you for reporting it. 47eb87752c1d8e2e0730a44df022f6bc18c8da16 is the change to address the issue.

By the way, we would remove the entry "あたらしいげんごう → 令和" since we no longer technically need it.
Best,
